sympy/assumptions/handlers/matrices.py

```diff
@@ -405,6 +405,38 @@ def DiagonalMatrix(expr, assumptions):
     Factorization = staticmethod(partial(_Factorization, Q.diagonal))
 
 
+class AskHermitianHandler(CommonHandler):
+
+    Identity, ZeroMatrix = [staticmethod(CommonHandler.AlwaysTrue)]*2
+
+    @staticmethod
+    def MatrixExpr(expr, assumptions):
```

Comment:
Currently, I have used `Identity`, `ZeroMatrix` and `MatrixExpr` subclass. Should I add for other classes like `MatrixSymbol`?